sklearn/datasets/_base.py

```diff
@@ -52,6 +52,11 @@ def get_data_home(data_home=None) -> str:
     data_home : str, default=None
         The path to scikit-learn data directory. If `None`, the default path
         is `~/sklearn_learn_data`.
+
+    Returns
+    -------
+    data_home: str
+        Path of the scikit-learn data directory.
```

Comment:
To be consistent the other doc-strings, I think it's better to have this description.
```suggestion
    data_home: str
        The path to scikit-learn data directory.
```